frontend/src/redux/configSlice.ts

```diff
@@ -9,6 +9,22 @@ export interface ConfigState {
   clusters: {
     [clusterName: string]: Cluster;
   } | null;
+  /**
+   * Stateless Clusters is a map of cluster names to cluster objects.
```

Comment:
Can you please answer these in the docs: 
- What is a stateless cluster? 
- Why is it separate from allClusters?

Reply:
Fixed it in the docs